frontend/src/__apps/parlamentar/main.js

```diff
@@ -78,7 +78,7 @@ const v = new Vue({ // eslint-disable-line
 
     pesquisaChange (event) {
       this.is_pesquisa = !this.is_pesquisa
-      this.filter_ativo = false
+      this.filter_ativo = true
```

Comment:
Eu não lembro dessa parte do código (foi o Ulysses que trabalhou nisso). Se colocar `true` isso afetaria o resultado da pesquisa? Tipo vir somente os parlamentares ativos? Desculpa a pergunta idiota.

Reply:
sim

Reply:
o filtro eh feito no fronte end, o be envia todo o conteudo

Reply:
O ideal seria que a pesquisa permitisse a gente recuperar quaisquer parlamentares, não somente os ativos, não?

Reply:
Acho que podemos deixar esse como false e colocar true somente na tela de listagem de parlamentar. O que acha?

Reply:
@edwardoliveira, na resposta da pesquisa é ignorado os valores das variáveis filter_ativo e filter_titular. Acho que está ok, falando só o build

Reply:
@joaohortsenado Quando vc puder, roda o `yarn build` pra gerar os assets, por favor, e incorpora nesse PR pra gente poder incluir essa mudança. 🙂

Reply:
tinha esquecido disso